Aptoide/aptoide-client-v8
# [ASV-297] Install Button And Login

**What does this PR do?**

Changed the layout for the not logged in install share click.
The implemented layout is available on : 
[Layout on Zeplin](https://app.zeplin.io/project/58a5dde3079da73c80cc892e/screen/5acddff65621757fb046d1f5)

On this layout, the don't show me again checkbox was not implemented since this layout is only shown 2 times (the 2nd and 4th time the user presses the install button).

**Database changed?**

No

**Where should the reviewer start?**

- [ ] not_logged_in_share.xml
- [ ] NotLoggedInFragment.java

**How should this be manually tested?**

If you are logged in, logout, go to the appview of any app, press install. On the 2nd and 4th time you press the install button you will be able to see this layout. After that, in order for you to see this layout, you must clear the cache.
**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-297](https://aptoide.atlassian.net/browse/ASV-297)

**Questions:**

   No



**Code Review Checklist**
